algopy_testing/src/algopy_testing/arc4.py

```diff
@@ -223,6 +271,123 @@ def __bool__(self) -> bool:
         return bool(self.native)
 
 
+_TDecimalPlaces = typing.TypeVar("_TDecimalPlaces", bound=int)
+_MAX_M_SIZE = 160
+
+
+class _UFixedNxMMeta(type(_ABIEncoded), typing.Generic[_TBitSize, _TDecimalPlaces]):  # type: ignore  # noqa: PGH003
```

Comment:
Can this ignore the specific type violations using `# type: ignore[RULE1, RULE2, ...]` instead of PGH003?

Reply:
It seems unnecessary and has been removed in this commit 2cacca23751e39919ec2ec76e26d927599c05f16.